benchmark/bench_CQRRP/CQRRP_single_precision.cc

```diff
@@ -105,7 +105,7 @@ static std::vector<long> call_all_algs(
         i == 0 ? t_geqrf_best = dur_geqrf : (dur_geqrf < t_geqrf_best) ? t_geqrf_best = dur_geqrf : NULL;
 
         // Clear and re-generate data
-        data_regen<T_rest, RNG>(m_info_rest, all_data_rest, state_gen, 0);
+        data_regen<T_rest>(m_info_rest, all_data_rest, state_gen, 0);
```

Comment:
``T_rest`` can be inferred from ``all_data_rest``.

Reply:
Fixed.